app/routes/groups_.$groupId.tsx

```diff
@@ -3,20 +3,132 @@ import { useParams, useLoaderData, Link } from '@remix-run/react';
+                    alt="Twitter Icon"
+                  />
+                </div>
+              </div>
+            </div>
+
+            <div className="flex place-content-evenly flex-wrap">
+              <div className="bg-white mt-2 text-black rounded-3xl px-4 flex flex-row items-center">
+                <div className="m-2 rounded-full h-8 w-8 justify-center items-center">
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708120241/avatar-astronaut_icc950.png"
+                    alt="Astronaut avatar"
+                  />
+                </div>
+                <p>315 Members</p>
+              </div>
+              <div className="bg-white mt-2 text-black rounded-3xl px-4 flex flex-row items-center">
+                <div className="m-2 rounded-full h-8 w-8 justify-center items-center">
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708121513/location_wf4qsz.png"
+                    alt="GPS Icon"
+                  />
+                </div>
+                <p>Location, USA</p>
+              </div>
+              <div className="bg-white mt-2 text-black hover:bg-warm hover:text-secondary rounded-3xl px-4 flex flex-row items-center">
+                <div className="m-2 rounded-full h-8 w-8 justify-center items-center">
+                  <Image
+                    className="w-full"
+                    src="https://res.cloudinary.com/dxctpvd8v/image/upload/v1708121700/share_hmihwe.png"
+                    alt="Arrowed share icon"
+                  />
+                </div>
+                <p>Share</p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="min-w-[280px] min-h-[280px] p-2 md:p-10 md:flex md:flex-row">
+          <div className="md:w-1/2 p-2 md:p-4">
+            <h2 className="font-extrabold md:text-2xl">What we are about</h2>
+            <p>{data.group.description}</p>
+          </div>
+          <div className="md:w-1/2 p-2 md:p-4 md:border-secondary md:border-2 rounded-lg">
+            <h2 className="font-extrabold md:text-2xl">Upcoming Events</h2>
+            <div className="flex flex-col">
+              {!data.group.events ? (
+                <div className="bg-white rounded-lg text-black p-2 min-h-[64px] my-2">
+                  <p>No Upcoming Events scheduled at this time</p>
+                </div>
+              ) : (
+                (data.group.events as any[]).map((item) => (
+                  <div key={item.id} className="bg-white rounded-lg text-black p-2 min-h-[64px] my-2">
+                    <p>{item}</p>
+                  </div>
+                ))
+              )}
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="py-32">
+        <Button variant="warm-block" buttonStyle="rounded" size="large">
```

Comment:
You can use the warm variant since we have merged the PR. 🎉

Reply:
Nice! This was done!